### ISSUE TITLE
TypeScript Intellisense very slow with some projects

### ISSUE THREAD

**Issue by dbaeumer** [Microsoft/TypeScript#9019]
_From @AmrN on June 7, 2016 19:19_

- VSCode Version: 1.2.0
- OS Version: Ubuntu 14.04

I'm using [angular2-webpack-starter](https://github.com/AngularClass/angular2-webpack-starter), and after cloning the repo and doing an `npm install` typescript intellisense is extremely slow.
One thing I've noticed is that the `exclude` list paths in **tsconfig.json** are written like this:

```json
{
  "exclude": [
    "./node_modules",
    "./typings/main.d.ts",
    "./typings/main"
  ]
}
```
But once I remove the dot slashes and restart VS Code things seem to be fixed and intellisense works fast:

```json
{
  "exclude": [
    "node_modules",
    "typings/main.d.ts",
    "typings/main"
  ]
}
```

I've reported the issue AngularClass/angular2-webpack-starter#660 but it seems the problem lies with VS Code since I've tried Atom with its typescript plugin and it works properly with that starter by default.

_Copied from original issue: Microsoft/vscode#7349_

**Comment by mhegazy** [comment]
@yuit can you take a look. looks related to https://github.com/Microsoft/TypeScript/issues/8998

**Comment by yuit** [comment]
I believe it is the same issue as #8998. We used to not recognize relative path used in "exclude" or "outdir" (which get include in "exclude" property") the fix is in the master already (PR #9025)
